src/sentry/templates/sentry/layout.html

```diff
@@ -32,6 +32,9 @@
 
   {% block scripts %}
   <script src="{% asset_url "sentry" "dist/vendor.js" %}"{% crossorigin %}></script>
+  {% ifnotequal request.LANGUAGE_CODE "en" %}
+  <script src="{% locale_js_url request %}"{% crossorigin %}></script>
```

Comment:
So, what if I just had a template tag that spat out `<script>` if there is a language code, and a supported language. Otherwise, it prints nothing.

i.e. `{% locale_js_include %}`